opennms-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -54,7 +54,7 @@
 
   <context-param>
     <param-name>parentContextKey</param-name>
-    <param-value>daoContext</param-value>
+    <param-value>restContext</param-value>
```

Comment:
Given that these are also used by some of the JSP and not exclusively for REST. Can we rename the context to `webContext`?